### ISSUE TITLE
can't render glyphs from ChordSymbol

### ISSUE THREAD

**Issue by AaronDavidNewman** [vexflow/vexflow#64]
In order to implement the chord editor in Smoosic, I used to be able to go:

``` typescript
new VF.Glyph( VF.ChordSymbol.glyphs['b'], 12);
```
For instance for the accidental flat.  But this doesn't work now - the chord symbol returns the unicode `'\ued60'` but the glyph code is expecting 'csymAccidentalFlat'.

**Comment by rvilarl** [comment]
`VF.Glyph` will disappear in the migration. The idea is to use `Element`

```
const el = new Element();
el.setText(ChordSymbol.glyphs['b']);
el.setFontSize(12);
```

**Comment by AaronDavidNewman** [comment]
How do I know the dimensions of the glyph?  I need this to format the preview.

**Comment by rvilarl** [comment]
You can use: GetTextMetrics GetWidth GetHeight

**Comment by ronyeh** [comment]
If you depend on the Glyph class directly, please let Rodrigo know. V5 may not need it internally, but if it is actually useful to people for important reasons, perhaps it is good to restore it?

**Comment by rvilarl** [comment]
@AaronDavidNewman did it work? If there is functionality from glyph that you miss, let me know.

**Comment by AaronDavidNewman** [comment]
For just rendering the music from the saved scores, I just need to know how to map what used to be the glyph names (e.g. csymAccidental) to the unicode that vexflow expects now.  I added those strings to the 'glyphs' structure in chordsymbol.ts in my own branch.  If required, I could move it into smoosic itself. 

To actually interact with the user, and to correctly format the music, I need to know the metrics from the glyph.  There is nothing to call 'GetTextMetrics ' on until I have created the vex objects, which I can't do until I have decided how to layout the music.  Not just for chord changes, if I want to know the width of a clef, for instance,  I'm used to getting the glyph metrics.  I could create the vex objects and save them until I need them, or create a cache for different objects, but I'm not set up to do that right now - I'd have to keep track of them all, or create them all a second time. 

So I'm not going to worry about this part.  I have a vex5 branch of smoosic to generate the test js files with the correct identifiers. I'll continue to develop new Smoosic features against version 4 for now, until there is a reason to switch.
